### ISSUE TITLE
About Cross Validation Experiments Results and Training Config

### ISSUE THREAD

**Issue by liao1fan** [PHOENIXcenter/pi-PrimeNovo#13]
I would like to use the cross-validation experimental dataset of DeepNovo to verify the cross-validation results of primeNovo. Currently, I am using yeast species as the test set and other species as the training set, but the training results are very poor. Could you provide the weights of the cross-validation experiment and **the COMPLETE config for training** the cross-validation experiment? If possible, please send them to my email: liaofanyishi1@163.com. Thank you very much!

**Comment by wyattz23** [comment]
It's very hard to train PrimeNovo on cross-validation 9species dataset, as 9species training data's quality is fairly low after our investigation. We didn't have a stable config that supports guaranteed PrimeNovo's training on yeast. It usually takes lots of trail (even random seed) to make the performance converge. But refinenove is opposite, which is ultra stable. Refer to RefineNovo's page for training NAT.
